spec/features/user_views_list_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+feature 'User views list' do
+  scenario "and see's the list's books" do
+    list = create(:list_with_books, books_count: 2)
+
+    visit lists_path
```

Comment:
@evan-007 @ajkamel It's unnecessary to start from root_path because the target might be deep in navigation to reach the page that we want to test and it will be slow. Another thing, you may end up with testing the same functionality multiple times.

What I usually do start from whatever path that makes sense, and here few examples:
* if I want to test to show a book detail page, so I think the best starting point should be books list page, then user should click on a specific book to see the book detail.
* If I want to test a user wants to give a review on book detail page, I will start with book detail page directly because we shouldn't need to test from books list page and user click to go to book detail page. We already cover it in the first scenario already.
* If we want to see books list page, it will make sense to start from root and click on navigation menu to see books list.